lib/stages/bufferMerge.hpp

```diff
@@ -29,14 +29,17 @@
  *
  * @par Buffers
  * @buffer in_bufs Array of input buffers to merge frames from.
+ *
```

Comment:
Not sure why there are extra spaces here?   It doesn't seem like they are intended.